ui/admin/app/styles/app.scss

```diff
@@ -1056,3 +1056,36 @@
     padding: 0.25rem 0.75rem;
   }
 }
+
+// worker filter generator
+.show-filter-generator {
+  .hds-form-field--layout-flag {
+    margin-bottom: 0;
+  }
+}
+
+.filter-generator-selection {
+  .hds-form-group__control-field {
```

Comment:
It seems like we're overriding our own style, is this right? We have a top level class just to override our own custom style?

Reply:
Yeah we have a global style that is applying margin-bottom and we needed remove it specifically for this one spot. We actually do it in a couple of places. So we have the `.filter-generator-selection` class to narrow the changes to just that spot.

Reply:
I think if we're starting to do this we should refactor this. We shouldn't be overriding our own styles as this becomes a nightmare to change due to specificity in the future.

Reply:
Sounds good, to be clear if the vast majority of the use case is valid in the global style, we should keep it. I wasn't sure how commonly we've been overriding this style at this point but if this has been more than just a few places, at the minimum we should make a utility class so we're not cluttering our CSS files just to override this same style as we're doing here.